app/controllers/resumes_controller.rb

```diff
@@ -0,0 +1,23 @@
+class ResumesController < ApplicationController
+  def index
+    render('layouts/application')
+  end
+
+  def create
+    resume = Resume.new(resume_params)
+    save = resume.save
+    if save and resume.data.attached?
+      Rails.logger.info('worked well')
+      render inline: "#{url_for(resume.data)}"

```

Comment:
this is only added for test (when uploading a resume it will return back the link to the active storage blob).